lib/plugins/aws/deploy/lib/createStack.js

```diff
@@ -69,9 +68,10 @@ module.exports = {
               // custom deployment bucket won't generate any output (no check)
               !this.serverless.service.provider.deploymentBucket;
             if (shouldCheckStackOutput) {
-              const isAlreadyAccelerated = !!_.find(data.Stacks[0].Outputs, {
-                OutputKey: 'ServerlessDeploymentBucketAccelerated',
-              });
+              const isAlreadyAccelerated = !!data.Stacks[0].Outputs.find(
```

Comment:
In this case, we can remove `!!` (boolean coercion) and replace `find` with `some`

Reply:
Nice catch.  I revised my PR based on your feedback.